docs/java-rest/high-level/indices/freeze_index.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[java-rest-high-freeze-index]]
```

Comment:
There is a newer style for doing these. search for `{upid}` in .asciidocs and you will see them.

Reply:
every doc in this entire folder is like this. I am keeping it that way.

Reply:
Good call, #35971 addresses it.

Reply:
merged and backported. <3